dev-python/os-api-ref/os-api-ref-2.2.0.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="8"
+
+PYTHON_COMPAT=( python3_{8..9} )
+
+inherit distutils-r1
+
+DESCRIPTION="Sphinx Extensions to support API reference sites in OpenStack"
+HOMEPAGE="https://docs.openstack.org/os-api-ref/latest"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	>=dev-python/openstackdocstheme-2.2.1[${PYTHON_USEDEP}]
+	>=dev-python/pyyaml-3.1.2[${PYTHON_USEDEP}]
+	>=dev-python/six-1.10.0[${PYTHON_USEDEP}]
+	dev-python/sphinx[${PYTHON_USEDEP}]
+"
+DEPEND="
+	${RDEPEND}
+	>=dev-python/pbr-2.0.0[${PYTHON_USEDEP}]
+	!~dev-python/pbr-2.1.0[${PYTHON_USEDEP}]
+
+	test? (
+		>=dev-python/beautifulsoup-4.6.0[${PYTHON_USEDEP}]
+		>=dev-python/sphinx-testing-1.0.1[${PYTHON_USEDEP}]
+		>=dev-python/stestr-2.0.0[${PYTHON_USEDEP}]
+		>=dev-python/subunit-1.0.0[${PYTHON_USEDEP}]
+		>=dev-python/testtools-2.2.0[${PYTHON_USEDEP}]
+	)
+"
+#		>=dev-python/testrepository-0.0.18[${PYTHON_USEDEP}]
+
+python_test() {
+	stestr run --exclude-regex 'os_api_ref.tests.test_microversions.TestMicroversions.test_parameters_table' || die
```

Comment:
Please don't use stestr if possible, it causes horrendous dependency cycles. Prefer pytest or unittest, though pytest doesn't work correctly for some stestr packages.

Reply:
pytest works fine